src/Admin/OptionsMenu.php

```diff
@@ -0,0 +1,490 @@
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * OptionsMenu constructor.
+	 *
+	 * @param GoogleFonts  $google_fonts An instance of the GoogleFonts service.
+	 * @param ReaderThemes $reader_themes An instance of the ReaderThemes class.
+	 */
+	public function __construct( GoogleFonts $google_fonts, ReaderThemes $reader_themes ) {
+		$this->google_fonts  = $google_fonts;
+		$this->reader_themes = $reader_themes;
+	}
+
+	/**
+	 * Adds hooks.
+	 */
+	public function register() {
+		add_action( 'admin_post_amp_analytics_options', 'AMP_Options_Manager::handle_analytics_submit' );
+		add_action( 'admin_menu', [ $this, 'add_menu_items' ], 9 );
+
+		$plugin_file = preg_replace( '#.+/(?=.+?/.+?)#', '', AMP__FILE__ );
+		add_filter( "plugin_action_links_{$plugin_file}", [ $this, 'add_plugin_action_links' ] );
+
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+	}
+
+	/**
+	 * Add plugin action links.
+	 *
+	 * @param array $links Links.
+	 * @return array Modified links.
+	 */
+	public function add_plugin_action_links( $links ) {
+		return array_merge(
+			[
+				'settings' => sprintf(
+					'<a href="%1$s">%2$s</a>',
+					esc_url( add_query_arg( 'page', AMP_Options_Manager::OPTION_NAME, admin_url( 'admin.php' ) ) ),
+					__( 'Settings', 'amp' )
+				),
+			],
+			$links
+		);
+	}
+
+	/**
+	 * Add menu.
+	 */
+	public function add_menu_items() {
+		/*
+		 * Note that the admin items for Validated URLs and Validation Errors will also be placed under this admin menu
+		 * page when the current user can manage_options.
+		 */
+		add_menu_page(
+			__( 'AMP Options', 'amp' ),
+			__( 'AMP', 'amp' ),
+			'manage_options',
+			AMP_Options_Manager::OPTION_NAME,
+			[ $this, 'render_screen' ],
+			self::ICON_BASE64_SVG
+		);
+
+		add_submenu_page(
+			AMP_Options_Manager::OPTION_NAME,
+			__( 'AMP Settings', 'amp' ),
+			__( 'General', 'amp' ),
+			'manage_options',
+			AMP_Options_Manager::OPTION_NAME
+		);
+
+		add_settings_section(
+			'general',
+			false,
+			'__return_false',
+			AMP_Options_Manager::OPTION_NAME
+		);
+
+		add_settings_section(
+			'validation',
+			false,
+			'__return_false',
+			AMP_Options_Manager::OPTION_NAME
+		);
+
+		add_settings_field(
+			Option::SUPPORTED_TEMPLATES,
+			__( 'Supported Templates', 'amp' ),
+			[ $this, 'render_supported_templates' ],
+			AMP_Options_Manager::OPTION_NAME,
+			'general',
+			[
+				'class' => 'amp-template-support-field',
+			]
+		);
+
+		/**
+		 * This fires when settings fields for the AMP Options menu need to be registered.
+		 *
+		 * This action is intended for internal use only, not to be used by other plugins.
+		 *
+		 * @internal
+		 */
+		do_action( 'amp_options_menu_items' );
+
+		$submenus = [
+			new AMP_Analytics_Options_Submenu( AMP_Options_Manager::OPTION_NAME ),
+		];
+
+		// Create submenu items and calls on the Submenu Page object to render the actual contents of the page.
+		foreach ( $submenus as $submenu ) {
+			$submenu->init();
+		}
```

Comment:
Note that we'll want to eliminate this as soon as we can. The analytics page should be moved under an “advanced” section on the main settings screen.